torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -404,25 +418,20 @@ c10::intrusive_ptr<RRef> RRefContext::delForkOfOwner(
   {
     std::lock_guard<std::mutex> lock(mutex_);
     auto rrefIter = forks_.find(rrefId);
-    TORCH_INTERNAL_ASSERT(
```

Comment:
Let's add a comment to explain that we don't do membership checking here because we want to support idempotency.